internal/distributed/proxy/service.go

```diff
@@ -240,6 +307,7 @@ func (s *Server) init() error {
 	s.etcdCli = etcdCli
 	s.proxy.SetEtcdClient(s.etcdCli)
 	s.wg.Add(1)
+	go s.startInternalGrpcLoop(Params.InternalPort)
```

Comment:
Why not invoke startInternalGrpLoop in Init method instead of startGrpcLoop method